samples/kubernetes/end2end/docs/01-pv-path.md

```diff
@@ -0,0 +1,35 @@
+## Configuring PV Path
```

Comment:
In this sample we need to create three PersistentVolumes (PV) and PersistentVolumeClaims(PVC) to store data for MySQL, Prometheus, and Grafana.   See the yaml files for the three PVs/PVCs.